app/controllers/historical_percent_changes_controller.rb

```diff
@@ -0,0 +1,29 @@
+class HistoricalPercentChangesController < ApplicationController
```

Comment:
I might consider controller and use case names like `AnnualRentUpdates` and `ViewAnnualRentUpdates`, given the results contain more than just a percentage change, and it might make it easier to decipher what the controller is for.